test/util/util.go

```diff
@@ -224,10 +224,12 @@ func WaitForInstanceReconciledGeneration(client v1beta1servicecatalog.Servicecat
 	)
 }
 
-// WaitForBindingCondition waits for the status of the named binding to
-// contain a condition whose type and status matches the supplied one.
-func WaitForBindingCondition(client v1beta1servicecatalog.ServicecatalogV1beta1Interface, namespace, name string, condition v1beta1.ServiceBindingCondition) error {
-	return wait.PollImmediate(500*time.Millisecond, wait.ForeverTestTimeout,
+// WaitForBindingCondition waits for the status of the named binding to contain
+// a condition whose type and status matches the supplied one but also returns
```

Comment:
Rephrase this comment so that it doesn't use "but". That is a remnant of when this was a separate function that did additional work beyond what `WaitForBindingCondition` did.